tests/integration/test_int.py

```diff
@@ -0,0 +1,454 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2025 Red Hat, Inc.
+
+"""
+Integration tests for validating that trestle-bot output is consumable by complytime
+"""
+import json
+import logging
+import os
+import pathlib
+import subprocess
+import sys
+from typing import Generator, Tuple, TypeVar
+
+import pytest
+from click import BaseCommand
+from click.testing import CliRunner, Result
+from git import Repo
+
+from tests.testutils import TEST_DATA_DIR, setup_for_catalog, setup_for_profile
+from trestlebot.cli.commands.sync_cac_content import (
+    sync_cac_catalog_cmd,
+    sync_content_to_component_definition_cmd,
+)
+
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+test_content_dir = TEST_DATA_DIR / "content_dir"
+
+T = TypeVar("T")
+YieldFixture = Generator[T, None, None]
+
+_TEST_PREFIX = "trestlebot_tests"
+
+
+@pytest.mark.slow
```

Comment:
Was the intention to ignore this test during the development and only execute in CI tests, for example? It doesn't seem a slow test, but maybe using the "integration" marker could be more aligned since it uses ComplyTime.